contents/tutorials/laravel-analytics.md

```diff
@@ -0,0 +1,307 @@
+---
+title: How to set up analytics in Laravel
+date: 2024-02-19
+author: ["lior-neu-ner"]
+tags: ['product analytics']
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+import EventsInPostHogLight from '../images/tutorials/laravel-analytics/events-light.png'
+import EventsInPostHogDark from '../images/tutorials/laravel-analytics/events-dark.png'
+import InsightLight from '../images/tutorials/laravel-analytics/insight-light.png'
+import InsightDark from '../images/tutorials/laravel-analytics/insight-dark.png'
+
+[Product analytics](/product-analytics) enable you to gather and analyze data about how users interact with your Laravel app. To show you how to set up analytics, in this tutorial we create a basic Laravel app, add PostHog, and use it to [capture events](/docs/product-analytics/capture-events) and [create insights](/docs/product-analytics/insights).
+
+## 1. Create a basic Laravel app
+
+To show you the basics of using PostHog analytics in your backend, we create a simple Laravel app that has two pages:
+
+1. A `login` page where a user can enter their name, email, and company name in form.
+2. A `home` page that has some text and a button.
+
+First ensure [PHP](https://www.php.net/manual/en/install.php) and [Composer](https://getcomposer.org/) are installed. Then, create a new Laravel project called `laravel-analytics`:
+
+```bash
+composer create-project laravel/laravel laravel-analytics
+cd laravel-analytics
+```
+
+Next, replace the code in `routes/web.php` to define our app routes:
+
+```php file=routes/web.php
+<?php
+
+use Illuminate\Support\Facades\Route;
+
+Route::get('/log-in', function () {
+    return view('login');
+});
+
+Route::post('/log-in', 'App\Http\Controllers\AuthController@login');
+
+Route::get('/home', function () {
```

Comment:
It can, but I wanted to make it so users will go first to the log-in screen. I suppose I can redo the login page to be `/` route. I'll do this for future similar tutorials